src/main/java/org/spongepowered/common/mixin/core/entity/player/MixinEntityPlayerMP.java

```diff
@@ -687,6 +693,23 @@ public void playSound(SoundType sound, SoundCategory category, Vector3d position
     }
 
     @Override
+    public void playRecord(Vector3i position, RecordType recordType) {
+        playRecord0(checkNotNull(recordType, "recordType"), position);
+    }
+
+    @Override
+    public void stopRecord(Vector3i position) {
+        playRecord0(null, position);
+    }
+
+    private void playRecord0(@Nullable RecordType recordType, Vector3i position) {
+        checkNotNull(position, "position");
+        final BlockPos pos = new BlockPos(position.getX(), position.getY(), position.getZ());
+        this.connection.sendPacket(new SPacketEffect(1010, pos,
```

Comment:
Extract `1010` to a constant somewhere, and use it here and any other occurrences.